experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -290,6 +290,20 @@ function blow_up_chart(W::AbsSpec{<:Field, <:MPolyRing}, I::MPolyIdeal;
     E = oscar.EffectiveCartierDivisor(Y, ID, trivializing_covering=domain(p_cov), check=false)
     set_attribute!(Y, :exceptional_divisor, E)
     set_attribute!(IPY, :exceptional_divisor, E)
+
+    # Cache the isomorphism on the complement of the center
+    p_res_dict = IdDict{AbsSpec, AbsSpecMor}()
+    for i in 1:ngens(I)
+      UW = PrincipalOpenSubset(W, gens(I)[i])
```

Comment:
Don't do `gens(I)[i]`, we just spent a lot of effort to get rid of these things
```suggestion
      UW = PrincipalOpenSubset(W, gen(I,i))
```
or
```suggestion
      UW = PrincipalOpenSubset(W, I[i])
```
or for that matter, it seems you set `g=gens(I)` above, so
```suggestion
      UW = PrincipalOpenSubset(W, g[i])
```